Pipfile

```diff
@@ -17,5 +17,11 @@ structlog = "*"
 pytest = "==2.8.2"
 "flake8" = "==3.4.1"
 pylint = "==1.7.4"
+astroid = "==1.5.3"
 pylint-mccabe = "==0.1.3"
 pylint-quotes = "==0.1.6"
+
+
+[requires]
+
+python_version = "3.6.3"
```

Comment:
Should we just be specifying the major.minor here and let the lock specify the patch version?